src/app/screens/History/components/NoteItem/styles.module.scss

```diff
@@ -0,0 +1,80 @@
+@import 'scss/variables/_sizes.scss';
+@import 'scss/variables/_fontSizes.scss';
+@import '_colors.scss';
+
+.noteItem_title {
+  font-size: $large !important;
+  margin-bottom: 10px;
+  max-width: 100%;
+  overflow: hidden;
+}
+
+.noteItem_container {
+  border-radius: 5px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+  justify-content: space-between;
+  margin: 6px 3px 6px 3px;
+  padding: 2px 16px;
+  transition: 0.3s;
+  width: 45%;
+}
+
+/* On mouse-over, add a deeper shadow */
+.noteItem_container:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+.noteItem_deleteButton {
+  align-self: flex-end;
+  background: none !important;
+  border: none !important;
+  box-shadow: none !important;
+  color: #999 !important;
+  height: 20px;
+  margin-right: -20px;
+  width: 20px;
+}
+
+.noteItem_deleteButton:active {
+  background-color: none !important;
+  box-shadow: none !important;
+}
+
+.noteItem_text_textArea {
```

Comment:
Extract the common styles properties into a class and then extend it whenever you need those properties

For example:
`font-size: $medium !important;`
`max-width: 100% !important;`
`min-height: 0 !important;`